README.md

```diff
@@ -197,6 +197,10 @@ style_edition = "2024"
 
 ## Tips
 
+* When using `rustfmt` directly in a Cargo project, Cargo will determine [`edition`](#rusts-editions) from `Cargo.toml`, but rustfmt will not, as it has its own configuration file, `rustfmt.toml`.
```

Comment:
I could see someone getting confused by this first sentence with all the clauses. My suggestion would be to remove it and keep the two sentences below, which are clearer and more direct.

```suggestion
```